tests/net/nanocoap_cli/nanocli_client.c

```diff
@@ -327,7 +327,7 @@ int nanotest_client_get_non_cmd(int argc, char **argv)
 {
     int res;
 
-    uint8_t response[CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT];
+    uint8_t response[coap_szx2size(CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT)];
```

Comment:
We should better avoid VLAs. If this is actually known at compile time, you could use the `DECLARE_CONSTANT` to force the compiler to not use VLAs.

Reply:
`coap_szx2size` is a macro, so this should be compile-time const.

Reply:
Ah, OK.

A pity it is lower case, otherwise this would be obvious.